Chefs/Views/SettingsPage.xaml

```diff
@@ -57,7 +57,8 @@
 								 TextChanging="TextBox_TextChanging">
 							<ut:ControlExtensions.Icon>
 								<PathIcon Data="{StaticResource Icon_Phone_Iphone}"
-										  Foreground="{ThemeResource PrimaryBrush}" />
```

Comment:
The fix for this should come from Material, we need to be using a Viewbox here: https://github.com/unoplatform/Uno.Themes/blob/7f3fbdd50e703c1cb086c8440694b98072374a3e/src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml#L820-L833

Like we do for other usages of IconExtensions: 

https://github.com/unoplatform/Uno.Themes/blob/7f3fbdd50e703c1cb086c8440694b98072374a3e/src/library/Uno.Material/Styles/Controls/v2/FloatingActionButton.xaml#L226-L234

https://github.com/unoplatform/Uno.Themes/blob/7f3fbdd50e703c1cb086c8440694b98072374a3e/src/library/Uno.Material/Styles/Controls/v2/AppBarButton.xaml#L86-L95

A Viewbox should auto scale the content to the constraints of the width/height of the box

Reply:
For now let's remove this change and just keep this PR for fixing the notifications